utilities/rpp-unittests/HIP_NEW/testAllScript.sh

```diff
@@ -18,7 +18,7 @@ cwd=$(pwd)
 
 # Input Images - Three images (224 x 224)
 DEFAULT_SRC_FOLDER_1="$cwd/../TEST_IMAGES/three_images_224x224_src1"
-DEFAULT_SRC_FOLDER_2="$cwd/../TEST_IMAGES/three_images_224x224_src2"
+DEFAULT_SRC_FOLDER_2="$cwd/../TEST_IMAGES/three_images_224x224_src1"
```

Comment:
Same comment here for all changes under the rpp-unittests / rpp-performancetests folders